amend-startup-msg.sh

```diff
@@ -5,6 +5,6 @@
 
 path=$1
 
-find $path -type f -name 'configuration.toml' | xargs \
+find $path -type f -name 'configuration.yaml' | xargs \
```

Comment:
@MonicaisHer have you tested this? It injects toml syntax in a yaml file.

Reply:
I forgot they have different syntax, so I didn't test it. Updated now.